### PR TITLE
DTSRD-3562 Swagger Publishing

### PR DESCRIPTION
**Pull request by mikebrownccd**
### Jira link
https://tools.hmcts.net/jira/browse/DTSRD-3562

### Change description
Swagger Publishing

### Checklist

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] Does this PR introduce a breaking change

### CODE REVIEW

File: .github/workflows/swagger.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: Publish Swagger Specs
+
+on:
+  push:
+    branches:
+      - master
+      - DTSRD-3562_Swagger_Publishing
```
Comment (lukasz-wolski) [inline .github/workflows/swagger.yml:7]: possibly remove branch to match other PRs

Comment (lukasz-wolski) [inline .github/workflows/swagger.yml:7]: possibly remove branch to match other PRs

Comment (mikebrownccd) [reply .github/workflows/swagger.yml]: removed

**Review by lukasz-wolski — APPROVED** [review]

**Review by SabinaHMCTS — APPROVED** [review]

File: .github/workflows/swagger.yml (hunk 1 above)
Comment (mikebrownccd) [reply .github/workflows/swagger.yml]: removed

**Pull request merged**